frame/example-offchain-worker/src/lib.rs

```diff
@@ -118,6 +118,40 @@ pub fn de_string_to_bytes<'de, D>(de: D) -> Result<Vec<u8>, D::Error>
 	Ok(s.as_bytes().to_vec())
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct CallData {
+    bytes: Vec<u8>,
+}
+
+impl CallData {
+    /// Creates new call ABI data for the given selector.
+    pub fn new() -> Self {
+        Self {
+            bytes: vec![REPORT_METRICS_SELECTOR[0], REPORT_METRICS_SELECTOR[1], REPORT_METRICS_SELECTOR[2], REPORT_METRICS_SELECTOR[3]],
```

Comment:
There is the selector copied here, and then ignored by the `params` method. This is very confusing.